Piwik.Tracker/PiwikTracker.cs

```diff
@@ -126,11 +126,11 @@ public class PiwikTracker
         /// <summary>
         /// Ecommerce item page view tracking stores item's metadata in these Custom Variables slots.
         /// </summary>
-        private const int CvarIndexEcommerceItemPrice = 2;
+        internal const int CvarIndexEcommerceItemPrice = 2;

```

Comment:
If tests are only implemented using the mocked http server, can you confirm these fields could remain private?